File: service/src/main/java/com/financial/service/service/KrBankServiceImpl.java
package com.financial.service.service;

import com.financial.service.config.KrBankProperties;
import com.financial.service.domain.KrBankData;
import com.financial.service.domain.KrBankSchema;
import com.financial.service.dto.KrBankRequest;
import com.financial.service.repository.KrBankDataRepository;
import com.financial.service.repository.KrBankSchemaRepository;
import java.util.List;
import org.springframework.beans.factory.annotation.Autowired;

class KrBankServiceImpl implements KrBankService {
    @Autowired
    private KrBankProperties krBankProperties;

    @Autowired
    KrBankDataRepository krBankDataRepository;

    @Autowired
    KrBankSchemaRepository krBankSchemaRepository;

    @Override
    public List<KrBankSchema> getSchema(KrBankRequest krBankRequest) {
        return krBankSchemaRepository.findAll();
    }

    @Override
    public List<KrBankData> getData(KrBankRequest krBankRequest) {
        return krBankDataRepository.findAll();
    }
}

File: src/main/kotlin/com/minssan9/financial/krbank/service/KrBankServiceImpl.kt
package com.minssan9.financial.krbank.service

import com.minssan9.financial.config.AppProperties
import com.minssan9.financial.krbank.domain.KrBankData
import com.minssan9.financial.krbank.domain.KrBankSchema
import com.minssan9.financial.krbank.dto.KrBankDto
import com.minssan9.financial.krbank.repository.KrBankDataRepository
import com.minssan9.financial.krbank.repository.krBankSchemaRepository
import org.springframework.beans.factory.annotation.Autowired

class KrBankServiceImpl : KrBankService {
    @Autowired
    private lateinit var appProperties: AppProperties

    @Autowired
    lateinit var krBankDataRepository: KrBankDataRepository

    @Autowired
    lateinit var krBankSchemaRepository: krBankSchemaRepository

    override fun getSchema(krBankRequest: KrBankDto.KrBankRequest): List<KrBankSchema>? {
        return krBankSchemaRepository.findAll()
    }

    override fun getData(krBankRequest: KrBankDto.KrBankRequest): List<KrBankData>? {
        return krBankDataRepository.findAll()
    }

}
